### DISCUSSION TITLE
New here

### DISCUSSION THREAD

**Discussion by OnyekachiSamuel** [mozilla-mobile/firefox-ios#13291] [General]
Hello, I am new to contributing to this project, I am an iOS Engineer. I have checked for the opened issues that are first time friendly but they have all been worked on. I would like to work on some tasks. 
Also, it will be great if issues that has been resolved are removed from the `contribution ok` kind of issues. I hope someone will get back to me as soon as possible. Thanks.

**Comment by lmarceau** [comment]
Hello! Some tasks that aren't closed are pending QA checks. They will be closed after they've been tested. Here are some of the issues from the `Contribution ok` list that can be worked on:
- https://github.com/mozilla-mobile/firefox-ios/issues/12507
- https://github.com/mozilla-mobile/firefox-ios/issues/12479
- https://github.com/mozilla-mobile/firefox-ios/issues/10055
- https://github.com/mozilla-mobile/firefox-ios/issues/6885
Feel free to ask on other tickets that aren't marked as `Contributor OK` if the ticket can be worked on. We try to mark them whenever we can, but sometimes contributor ok tickets aren't always noted down.

**Reply by OnyekachiSamuel** [reply]
@lmarceau Thank you very much for your response, I will take a look at the issues you listed and get started.

**Reply by ivan-pixel-rgb** [reply]
@lmarceau Thanks for sharing these good points I noticed I'm also new here so please update us and share more helpful information for our website [hyenakey.io](https://www.insigniader.com)
thanks
